stipendien/goingabroad.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Going Abroad
+subtitle: Hier gibt es Informationen zu unseren Auslandsprogrammen.
+layout: page-custom
+---
+<div class="box" markdown="1">
+### Going Abroad Purdue
+
+#### Going Abroad Purdue, was soll das sein?
+Going Abroad Purdue ist ein Auslandsprogramm von PeP et. al. e.V. und der Arbeitsgruppe von
+[Andreas Jung](https://www.physics.purdue.edu/people/faculty/anjung.php).
+In den drei Monaten des Auslandsprogramms wirst Du Teil von Jungs Team an der
+[Purdue University](https://www.purdue.edu/), [West Lafayette](https://en.wikipedia.org/wiki/West_Lafayette,_Indiana)
+in den USA. Dort wirst Du an aktueller Forschung im Bereich der Hochenergiephysik beteiligt sein. Neben neuen
+Bereichen der Physik hast Du die Chance das amerikanische Leben kennenzulernen. Das Programm wird als Stipendium angeboten, das heißt ein Großteil der anfallenden Kosten wird übernommen.
+
+#### Purdue? Was ist das für eine Uni?
+Die [Purdue University](https://en.wikipedia.org/wiki/Purdue_University) ist eine Universität in West Lafayette,
+Indiana, USA. Sie ist als forschungsintensive Universität Mitglied der Association of American Universities.
+Etwa 40.000 Studenten besuchen die Universität und werden dabei von mehr als 3.000 Lehrkräften betreut.
+Übrigens, der erste und letzte Mensch auf dem Mond sind Purdue Absolventen.
+
+#### Wann findet das Auslandsprogramm statt?
+Das Auslandsprogramm findet vom 15.05.2018 bis zum 15.08.2018 statt.
+
+#### Wie und wo lebt man während des Aufenthalts?
+Während Deines Forschungsaufenthaltes lebst Du im Studentenwohnheim der Universität.
+Dort ist ein Bett in einem Doppelzimmer für Dich reserviert. 
+Du wohnst also mit einem anderen Studenten zusammen.
+
+Während Deines Forschungsaufenthalts arbeitest Du in etwa 8 Stunden pro Tag.
+Neben der Arbeit kannst Du in der Uni eigenen Mensa essen oder im Freizeitzentrum der Uni entspannen.
+Am Wochenende kannst Du zum Beispiel Chicago besichtigen oder am Lake Michigan wandern gehen.
+
+Nach Deinem Forschungsprojekt hast Du die Möglichkeit weitere 30 Tage in den USA zu verbringen.
+In dieser Zeit steht es Dir frei das Land auf eigene Kosten zu erkunden.
+Unter Absprache mit den Förderern kannst Du Dir den Ort des Rückflugs selbst aussuchen.
+
+#### Mit welchem Themengebiet beschäftigst Du dich?
+Das Programm findet im Bereich der Hochenergiephysik statt.
+
+Im Wesentlichen beschäftigst Du Dich dann mit diesem Themenbereich:
```

Comment:
Das liest sich etwas komisch. Vielleicht kann man die Beschreibung des Projekts in einen eigenen teil stecken.